deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -12,10 +12,9 @@ clusterName:
 # -- Optional override of the target kubernetes version
 kubeVersion:
 
-# -- Cluster ceph.conf override
-configOverride:
-# configOverride: |
-#   [global]
+# -- Cluster ceph.conf override - default value initialises empty config map per rook issue #11302
+configOverride: |
```

Comment:
I understand now the issue in #11302, thanks for the explanation in that issue. But now I'm wondering if we can even change this default. This looks good for new clusters who will later be upgraded. However, what about existing clusters that are not upgraded to this change? This seems like it would mean everyone will hit the issue described in #11302 if we go ahead with this change. 

We may need to go with the approach describe in [this comment](https://github.com/rook/rook/issues/11302#issuecomment-1329991009) so that the operator would add the helm ownership labels when it creates the configmap [here](https://github.com/rook/rook/blob/fcf634e3b7808e4338e7fd261c047aae4793e99b/pkg/operator/ceph/cluster/utils.go#L39). Just ignore my incorrect comments on the repro steps there.

On a related note, the operator would also need to update that configmap if it doesn't have the labels. Currently that code path only creates the configmap, but never updates it.

Reply:
"operator would add the helm ownership" - One of the annotations needed is: "meta.helm.sh/release-name: ...", so this won't work for people who customise the helm release name, unless operator somehow discovers the custom name. 

I can understand this PR might be too painful. While it won't overrite existing CMs (and any custom config people have added), it will cause "_Helm upgrade failed: rendered manifests contain a resource that already exists_" error on existing clusters, per #11302. Fix is simply to delete the CM, but can understand if you'd prefer to leave this broken. 

If so, worth documenting how to workaround and use configOverride. Choice is:
1. Add override settings you need or a dummy heredoc before you deploy the cluster.
2. If you missed #1, delete the existing rook-created cm.

Reply:
What is the release name value you see for the resources? I don't have a test cluster handy. If it's the version, the operator could just set it to the rook version, which we have as the same version as the helm chart in each release. If we can make this work, it will just avoid the need to document a workaround or require a workaround during upgrades.

Reply:
Not quite sure what you are asking. Release name from the main doc is [rook-ceph-cluster](https://github.com/rook/rook/blob/master/Documentation/Helm-Charts/ceph-cluster-chart.md#release). But like I say, people are free to name helm releases whatever they like, or (I assume) have multiple releases installing clusters.

Normally, the best way to solve this would be to name each resource after the release, that way they're all tied to the release and don't break each other. But I'm not sure how much operator code would need changing if we shifted away from the current fixed names. Whatever the solution is, it's not this PR, so closing.

Reply:
Ok, makes sense that the release would be `rook-ceph-cluster`. The operator should be able to set that label if it's missing since the `rook-ceph` chart doesn't ever create that configmap.